selfdrive/common/gpio.h

```diff
@@ -17,5 +17,9 @@
   #define GPIO_STM_BOOT0        0
 #endif
 
+enum EDGE_TYPES { rising, falling, both, none };
```

Comment:
```suggestion
enum Edgetypes { Rising, Falling, Both, None };
```

We're not 100% consistent in our codebase with enums lower/uppercase, but let's do this.

Reply:
sure